client/src/components/WelcomeHeader/WelcomeHeader.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./WelcomeHeader.css";
+
+// Created by Tyler. Not sure what font we're going with yet so feel free to adjust that and play around with the font size.
+const WelcomeHeader = props => {
+  // Default Class to apply to Component
+  let classList = `welcome-header`;
+
+  return <div className={classList}>Welcome, {props.name}!</div>;
```

Comment:
If you have this render children (i.e. `{children}`), then this could handle #52, #71, and #97 

That would modify the use slightly. You would render:

```jsx
<Header>
Welcome, {this.props.user.name}
</Header>
```

But then for #97, we could do something like this:

```jsx
<Header>
  <h3>Trip: Hawaii 2020</h3>
</Header>
```

and for #71, we could do this:

```jsx
<Header>
  <h3>User Profile</h3>
  <SearchInput placeholder={"Search My Directory"} onSubmit={this.handleSumbit} />
</Header>
```